lib/Runtime/Debug/TTSerialize.cpp

```diff
@@ -1814,8 +1833,8 @@ namespace TTD
     TraceLogger::TraceLogger(FILE* outfile)
         : m_currLength(0), m_indentSize(0), m_outfile(outfile)
     {
-        this->m_buffer = (char*)CoTaskMemAlloc(TRACE_LOGGER_BUFFER_SIZE);
-        this->m_indentBuffer = (char*)CoTaskMemAlloc(TRACE_LOGGER_INDENT_BUFFER_SIZE);
+        this->m_buffer = (char*)malloc(TRACE_LOGGER_BUFFER_SIZE);
```

Comment:
Malloc is not throwing- what is the right thing to do here if we OOM? terminate? throw?